server-src/src/main/java/com/itt/kmt/controllers/UserController.java

```diff
@@ -60,26 +62,42 @@ public ModelMap getUser(@PathVariable("id") final String id) {
     }
     /**
      * REST Interface for users retrieval.
+     * @param request request sent.
+     * @param response response received.
      * @return ModelMap.
+     * @throws Exception.
      */
     @RequestMapping(method = RequestMethod.GET)
     @RequiresPermissions("getAllUser")
-    public ModelMap getAllUsers() {
+    public ModelMap getAllUsers(final HttpServletRequest request, final HttpServletResponse response) {
+
+        String jwtToken = request.getHeader("Authorization");
+        User loggedInUser = userService.getLoggedInUser(jwtToken);
+
         List<User> users = userService.getAllUsers();
+        users.remove(loggedInUser);
+
         return new ModelMap().addAttribute("users", users);
     }
     /**
      * REST Interface for Admin and Managers retrieval.
+     * @param request request sent.
+     * @param response response received.
      * @return ModelMap.
      */
     @RequestMapping(value = "/approvers", method = RequestMethod.GET)
     @RequiresPermissions("getAllApprovers")
-    public ModelMap getAllApprovers() {
+    public ModelMap getAllApprovers(final HttpServletRequest request, final HttpServletResponse response) {
+
+        String jwtToken = request.getHeader("Authorization");
+        User loggedInUser = userService.getLoggedInUser(jwtToken);
+
         List<User> adminAndmanager = new ArrayList<User>();
```

Comment:
Have you ran formatter?, adminAndmanager should be name properly adminAndManager.

Reply:
Yes we are following check styles for coding guideline. But yes this should have been adminAndManager which i missed.
Will make the correction.